calcom/cal.com
# fix: remove edit location for non-in-person event types

## What does this PR do?


Hides edit location option while location type is not in-person event.

Fixes #1716 

## Type of change




## How should this be tested?



- It should only open the edit location modal with address input when in-person event location is selected
